presets.js

```diff
@@ -0,0 +1,170 @@
+export function getPresetDefinitions() {
+	const presets = {
+		preset_esc: {
+			type: 'button',
+			category: 'PPT',
+			name: 'Send ESC Command',
```

Comment:
Other modules try to explain what the action does. I can only guess what the ESC command does. "Abort current presentation/video"?

Also the other action names might benefit from a more user-oriented naming.
- "Show next/previous slide"
- "Terminate PPT"
- "Stop Video" (???)
- "Play Video 1"